src/js/ui/simply-pebble.js

```diff
@@ -140,7 +207,43 @@ var colorMap = {
 };
 
 var Color = function(color) {
-  return colorMap[color] ? colorMap[color] : colorMap.clear;
+  if (color.charAt(0)=='#') {
+      //Convert full hex to shorthand
+      if (color.length==4){
+          var r = color.charAt(1);
+          var g = color.charAt(2);
+          var b = color.charAt(3);
+          color = '#'+r+r+g+g+b+b;
+      }
+      //Ensure upper case
+      color = color.toUpperCase();
+      return hexColorMap[roundColor(color)];
+  }
+  return namedColorMap[color] ? namedColorMap[color] : namedColorMap.clear;
+};
+
+var roundColor = function (color) {
+    var closestTo = function(color, colors) {
```

Comment:
Use 2 spaces for an indent. This includes the `hexColorMap`. I apologize that the `namedColorMap` is not correctly indented, I didn't want to enforce strict formatting for someone's first contribution.